### ISSUE TITLE
Propagate font feature settings from editor to suggest widget

### ISSUE THREAD

**Issue by AlsoScratch** [microsoft/vscode#87994]
- VSCode Version: 1.41.1
- OS Version: macOS 10.15.2

Steps to Reproduce:

1. Open a file, such as the `settings.json` document
2. Type something that will make the fill suggestions appear

Does this issue occur when all extensions are disabled?: Yes

<img width="499" alt="Screenshot 2020-01-02 at 10 24 38 PM" src="https://user-images.githubusercontent.com/48752273/71671702-aee4a180-2dae-11ea-9af2-cf1b3e120571.png">

The text in the editor is correct based on my OpenType settings, but the fill suggestions are still the default. (Note differences in `i` and `l`)

**Comment by jrieken** [comment]
Are you talking about the font?

**Comment by AlsoScratch** [comment]
Of course, it is about the font.

**Comment by AlsoScratch** [comment]
Umm... I don't understand the new title. I just thought that when OpenType customization was implemented for the editor, the fill suggestions were overlooked.

**Comment by AlsoScratch** [comment]
Notice the `i` in the editor does not match the `i` in the fill suggestion list.

**Comment by jrieken** [comment]
> Umm... I don't understand the new title. I just thought that when OpenType customization was implemented for the editor, the fill suggestions were overlooked.

Forgot a word, updates the title again

**Comment by AlsoScratch** [comment]
<img width="544" alt="Screenshot 2020-01-03 at 4 47 07 PM" src="https://user-images.githubusercontent.com/48752273/71714544-44347400-2e49-11ea-9b56-4e0d8b3d6fd3.png">

**Comment by AlsoScratch** [comment]
The IntelliSense just needs to automatically match the editor.

**Comment by jrieken** [comment]
Well, turns out we actually do apply the editor font in the suggest widget. I guess this is a ligature issue. @AlsoScratch Please share your settings with me.

**Comment by AlsoScratch** [comment]
It's a font called [Iosevka](https://typeof.net/Iosevka/) and it uses what I may call _custom_ OpenType tags to modify individual characters like `g`, `i`, `l`, `y`, etc.

**Comment by AlsoScratch** [comment]
The `settings.json` file

```
{
    "editor.fontFamily": "'Iosevka Term', 'IBM Plex Mono', Menlo, Monaco, 'Courier New', monospace",
    "editor.fontLigatures": "'cv01', 'cv05', 'cv11', 'cv19', 'cv21', 'cv23', 'cv26', 'cv27', 'cv36', 'cv38', 'cv45', 'cv46', 'cv49', 'cv51', 'cv52', 'cv65'",
    "editor.lineHeight": 22
}
```

**Comment by jrieken** [comment]
Ok, I am starting to understand now (sorry not a font expert). I have pushed a change that makes sure that we use the same font features in the suggest widget as we do in the editor. Tomorrow's insider build should contains this change, please report back if it then works for you.

**Comment by AlsoScratch** [comment]
I don't use insider, I only trust official releases.

**Comment by jrieken** [comment]
Insiders is an official release.

**Comment by AlsoScratch** [comment]
> Insiders is an official release.

Isn't it the one that always has the latest commit, something like that?

**Comment by jrieken** [comment]
It's a daily build (usually from 8am CET) which contains the latest changes and allows you to see what will come in VS Code.

**Comment by AlsoScratch** [comment]
Oh, I just realized that the word 'settings' in the title is misspelled.

**Comment by octref** [comment]
<img width="435" alt="Screen Shot 2020-01-08 at 10 35 10 AM" src="https://user-images.githubusercontent.com/4033249/71966870-a6eb8e00-3202-11ea-8b9a-dec607024287.png">

@jrieken Your change is applied to the details widget, but I think the OP is asking for the suggest item list to share font feature setting.

I don't think we want to apply the font feature settings to the details widget, as it uses system's default sans-serif font (such as San Francisco on macOS), and font features that apply to monospace coding font (such as his choice, Ioveska) might cause weird rendering in details widget.

In short, I suggest taking https://github.com/microsoft/vscode/commit/6f428b15103d05bd2660ae26b029f4417eaedd79 and removing https://github.com/microsoft/vscode/commit/bf2dbfc3412c69bea9a5f1c893deb046316bb7b0.

**Comment by octref** [comment]
@jrieken Lemme know if I can remove https://github.com/microsoft/vscode/commit/bf2dbfc3412c69bea9a5f1c893deb046316bb7b0

**Comment by AlsoScratch** [comment]
It also needs to be propagated to the Python terminal.

**Comment by octref** [comment]
Please open new issue for terminal then. It is a completely different area.
